docker-bake.hcl

```diff
@@ -7,6 +7,9 @@ variable "QEMU_REPO" {
 variable "QEMU_VERSION" {
   default = "v6.2.0"
 }
+variable "QEMU_PATCHES" {
```

Comment:
this is more like `BUILDKIT_PATCHES`, should probably not be reused for `src`

Reply:
Do you want me to s/QEMU_PATCHES/BUILDKIT_PATCHES` ? I need patches in the src. If you prefer, I can keep `src` like mainline, and do an `src-patched` with the patches.

Reply:
if you need buildkit patches you should either define `buildkit-src` target or use `--set`

Reply:
Variable name still not correct